.circleci/config.yml

```diff
@@ -383,74 +301,98 @@ workflows:
       - test
 
       - build-android-release:
-          context: mattermost-mobile
+          context: mattermost-mobile-android-release
           requires:
             - test
           filters:
             branches:
-              only: /^build-(?!ios|pr|number).*/
+              only:
+                - /^build-\d+$/
+                - /^build-android-\d+$/
+                - /^build-android-release-\d+$/
       - deploy-android-release:
-          context: mattermost-mobile
+          context: mattermost-mobile-android-release
           requires:
             - build-android-release
           filters:
             branches:
-              only: /^build-(?!ios|pr|number).*/
+              only:
+                - /^build--\d+$/
```

Comment:
the double dash is intentional? i see there are other with just 1

Reply:
great catch